pkg/notifications/shoutrrr.go

```diff
@@ -240,10 +243,15 @@ func getShoutrrrTemplate(tplString string, legacy bool) (tpl *template.Template,
 	return
 }
 
+// StaticData is the part of the notification template data model set upon initialization
+type StaticData struct {
+	Title string
+	Host  string
+}
```

Comment:
Could this naming be improved to clarify what it is? `StaticData` is very generic.

Reply:
It is, but it's the best way I found to describe it. It's the non-changing (static) part of the data used for notifications. It could perhaps be called `StaticTemplateData`, but it's *strictly* not only used for that.
Any suggestions?